src/pulseview/pv/data/decoderstack.cpp

```diff
@@ -387,8 +387,15 @@ void DecoderStack::decode_proc()
 
 	assert(segment_);
 
-	// Prevent any other decode threads from accessing libsigrokdecode
-	lock_guard<mutex> srd_lock(global_srd_mutex_);
+    // Prevent any other decode threads from accessing libsigrokdecode
```

Comment:
whitespace is inconsistent with the rest of the file

Reply:
Indeed. I played with the settings of my IDE for another project, and forgot about that. Everything should be fine now. Thanks for noticing!